doc/nrf/ug_nrf5340.rst

```diff
@@ -567,6 +567,44 @@ See the following instructions.
    :start-after: fota_upgrades_start
    :end-before: fota_upgrades_end
 
+Simultaneous multi-image DFU
+****************************
+
+The simultaneous update of multiple images is available for testing since |NCS| v1.7.0.
+It allows the updating of both the application core and the network core in one go.
+
+To enable the simultaneous update of multiple images in the MCUboot, set the following options:
+
+* :kconfig:`CONFIG_BOOT_UPGRADE_ONLY` - The simultaneous update of multiple images does not support network core image reversion, so you need to disable application image reversion.
+* :kconfig:`CONFIG_PCD_APP` - Enable commands exchange with the network core.
+* :kconfig:`CONFIG_UPDATEABLE_IMAGE_NUMBER` - Enable support for multiple update partitions by setting this option to ``2``.
+
+To enable the simultaneous update of multiple images in the application, in addition to enabling the MCUboot support, set the following options:
+
+* :kconfig:`CONFIG_UPDATEABLE_IMAGE_NUMBER` - Enable support for multiple update partitions by setting this option to ``2``.
```

Comment:
Is the same option supposed to be in both places?

Reply:
yes